theano/tensor/nnet/tests/test_sigm.py

```diff
@@ -306,7 +306,6 @@ def test_grad_log1msigm(self):
         mode = self.get_mode()
         if not isinstance(mode, theano.compile.DebugMode):
             f = theano.function([x, lr], ux, mode=mode)
```

Comment:
I was confused, I thought it was another part of the code.
This is fine like it is.